api/client/build.go

```diff
@@ -313,13 +327,24 @@ func (cli *DockerCli) CmdBuild(args ...string) error {
 		if jerr.Code == 0 {
 			jerr.Code = 1
 		}
+		if *suppressOutput {
+			return Cli.StatusError{
```

Comment:
Like above, I think this entire return can be replace with:
```
fmt.Fprintf(cli.err, "%s%s", outBuf, sopts.out)
```
and then let it fall thru to the return below.